.circleci/config.yml

```diff
@@ -51,7 +52,12 @@ jobs:
 
       # Run UI e2e tests
       #- run: npm run cypress -- run --browser chrome --spec "cypress/integration/(Regression).js" --record --key $CYPRESS_RECORD_KEY
-      - run: npm run cypress -- run --spec "cypress/integration/(Regression).spec.js" --record --key $CYPRESS_RECORD_KEY
+      - run: npm run cypress -- run --record --key $CYPRESS_RECORD_KEY --parallel --spec "cypress/integration/CreateModels/AllEntityTypes.spec.js,cypress/integration/CreateModels/DisqualifyingEntities.spec.js,cypress/integration/CreateModels/Travel.spec.js,cypress/integration/CreateModels/WaitVsNoWaitActions.spec.js,cypress/integration/CreateModels/WhatsYourName.spec.js,cypress/integration/EditAndBranching/AddEndSessionAction.spec.js,cypress/integration/EditAndBranching/Branching.spec.js,cypress/integration/EditAndBranching/VerifyEditTrainingControlsAndLabels.spec.js,cypress/integration/ErrorHandling/ActionUnavailable.spec.js,cypress/integration/ErrorHandling/MissingAction.spec.js,cypress/integration/ErrorHandling/TwoConsecutiveUserInput.spec.js,cypress/integration/ErrorHandling/WaitNonWait.spec.js,cypress/integration/Log/WhatsYourName.spec.js,cypress/integration/Train/BookMeAFlight.spec.js,cypress/integration/Train/DisqualifyingEntities.spec.js,cypress/integration/Train/MyNameIs.spec.js,cypress/integration/Train/ScenarioAndTags.spec.js,cypress/integration/Train/TagAndFrog.spec.js,cypress/integration/Train/WaitVsNoWaitActions.spec.js,cypress/integration/Train/WhatsYourName.spec.js,cypress/integration/UX/BotModelMismatch.spec.js"
```

Comment:
Why does it need to be written like this?

It's difficult to read and difficult to maintain

Reply:
These are changes in progress. I needed to test CircleCI and the only way I knew of doing that was by actually submitting a PR. It was not triggering build when I pushed changes into my branch. All of the garbage will be removed before final PR.

I thought if I did not assign you as a reviewer that you would ignore this.